.pre-commit-config.yaml

```diff
@@ -79,15 +79,15 @@ repos:
     hooks:
       - id: rst-backticks
         name: "Check RST: No single backticks"
-        files: '^pep-\d+\.(rst|txt)$'
+        files: '^peps/pep-\d+\.rst$'
         types: [text]
```

Comment:
Let's set this to `types: [rst]`, which is the default so we can remove it, and also it on all RST files by removing `files:`

```suggestion
```